Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM registry.opensource.zalan.do/stups/openjdk:1.8.0-222-22
+FROM registry.opensource.zalan.do/stups/openjdk:1.8.0-232-23
 MAINTAINER Team Aruha, team-aruha@zalando.de
 
-ENV KAFKA_VERSION="2.3.0" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.6.2"
+ENV KAFKA_VERSION="2.4.1" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.6.2"
 ENV KAFKA_LOGS_DIR="/data/logs"
 ENV KAFKA_DIR="/opt/kafka"
 ENV HEALTH_PORT=8080
 ENV KAFKA_SETTINGS="${KAFKA_DIR}/config/server.properties"
 ENV BUKU_FEATURES="restart_on_exhibitor,rebalance_on_start,graceful_terminate,use_ip_address"
 ENV KAFKA_OPTS="-server -Dlog4j.configuration=file:${KAFKA_DIR}/config/log4j.properties -Dkafka.logs.dir=${KAFKA_LOGS_DIR} -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=32M -javaagent:/opt/jolokia-jvm-agent.jar=host=0.0.0.0"
-ENV KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false"
+ENV KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=localhost -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.rmi.port=8004 -Dcom.sun.management.jmxremote.port=8004"
```

Comment:
@ferbncode do we need to keep this jmx configs used during profiling?

Reply:
I am also not sure if we need this, @ferbncode

Reply:
Yes, sorry, I missed these comments. We do not require this configs, they are only useful when profiling single broker. I'll remove them.

Reply:
@rcillo Please do let me know if I should open a fix for this or you'd like to remove the parameters as part of this PR. o/

Reply:
@ferbncode I will remove it. As for the exposed ports, 8004 is the jmx one but we access jmx through jolokia so I just keep 8778.

Reply:
@ferbncode @adyach https://github.com/zalando-nakadi/bubuku/pull/147/commits/7305066a9aec613ccc5cc426706fc56c06ab64d3